### PR TITLE
REVIEW ONLY PR -- WILL NOT MERGE

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

File: confidential-big-data-spark/README.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+
+> As a pre-requisite to this scenario, an Azure SQL Database with Always Encrypted with secure enclaves is required. Please follow the detailed steps outlined in this [`sql-server-samples` repository](https://github.com/microsoft/sql-server-samples/blob/master/samples/features/security/always-encrypted-with-secure-enclaves/azure-sql-database/README.md) to deploy the `ContosoHR` Database. 
+
+Our Spark application will process 2 sample datasets from 2 data sources:
+1. [Azure Data Lake Storage - Parquet files](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction): It is a publicly available common [NYC Taxi](https://docs.microsoft.com/en-us/azure/open-datasets/dataset-taxi-yellow?tabs=pyspark) Dataset - where we demonstrate a simple Spark Job ([`COUNT *`](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.DataFrame.count.html)) on 1.5 Billion Rows of Parquet files (50 GB) stored on Azure Data Lake Storage. The purpose here is to show the horizontal scalability (multiple-pods perform a single job) enabled through this architecture.
```
Comment (magrande) [inline confidential-big-data-spark/README.md:30]: Can you remove the localization segment (en-us) in all the links ?

File: confidential-big-data-spark/README.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+2. [Azure SQL DB - Always Encrypted with secure enclaves](https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/always-encrypted-enclaves?view=sql-server-ver15): We showcase how to access Always Encrypted data (Encrypted [`ContosoHR` Database](https://github.com/microsoft/sql-server-samples/blob/master/samples/features/security/always-encrypted-with-secure-enclaves/azure-sql-database/README.md#demos-steps-1) from this sample) as plaintext inside the Spark Container Enclave. This service extends confidentiality from compute infrastructure to Azure PaaS service to achieve end to end solution confidentiality.
```
Comment (magrande) [inline confidential-big-data-spark/README.md]: Can you remove the localization segment (en-us) in all the links ?

File: confidential-big-data-spark/README.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+> 💡 Please note that SCONE (Scontain UG) is an Azure partner in confidential computing space who enables easy support to existing docker containers on Intel SGX Enclaves. You can also choose from other OSS and partners from [here](https://docs.microsoft.com/en-us/azure/confidential-computing/confidential-containers).
+
+   - **Sconedocs**: Scone's official documentation for Azure integration and getting started: [link](https://sconedocs.github.io/aks/)
+
+### Scenario
+
+In this scenario, we leverage Spark with Scone on a [Confidential Capable AKS cluster](https://docs.microsoft.com/en-us/azure/confidential-computing/confidential-nodes-aks-get-started) to showcase a sample pattern for processing larger datasets in a distributed fashion, as well as showcasing confidential analytics on relational Database Engines storing confidential data.
```
Comment (magrande) [inline confidential-big-data-spark/README.md]: This link is broken. This is the working one:

https://docs.microsoft.com/azure/confidential-computing/confidential-enclave-nodes-aks-get-started

File: confidential-big-data-spark/README.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+3. Get access to the PySpark base image used in this demo from Scone's [Gitlab](gitlab.scontain.com) Container Registry: `registry.scontain.com:5050/community/spark:5.6.0plus` - see [instructions here for Gitlab access](https://sconedocs.github.io/SCONE_Curated_Images/).
+4. Deploy the `ContosoHR` demo database outlined [in this sample](https://github.com/microsoft/sql-server-samples/blob/master/samples/features/security/always-encrypted-with-secure-enclaves/azure-sql-database/README.md) and configure a Service Principal to have the following *Key Permissions* on the Key Vault storing the [CMK](https://docs.microsoft.com/en-us/azure/azure-sql/database/always-encrypted-azure-key-vault-configure?tabs=azure-powershell#master-key-configuration):
+ - Key Management Operations: `Get`, `List`
+ - Cryptographic Operations: `Unwrap Key`, `Wrap Key`, `Verify`, `Sign`
+
+**Azure Infrastructure deployment - PowerShell:**
```
Comment (magrande) [inline confidential-big-data-spark/README.md:66]: Should the Storage account / Blob be provisioned in this step, or how is it provisioned ?

**Review by magrande — COMMENTED** [review]
Looks good, please see my comments

File: confidential-big-data-spark/README.md (hunk 4 above)
Comment (agowdamsft) [reply confidential-big-data-spark/README.md:66]: there is no storage in the deployment I read it will check again.

Comment (magrande) [reply confidential-big-data-spark/README.md:66]: @agowdamsft , I believe that Storage Account is used in NYC Taxi & Limousine Commission - yellow taxi trip records. So if you confirm you don't own that provisioning I will resolve this conversation

Comment (agowdamsft) [reply confidential-big-data-spark/README.md:66]: Its stored in a shared location as referenced here https://docs.microsoft.com/en-us/azure/open-datasets/dataset-taxi-yellow?tabs=azureml-opendatasets#storage-location

**Review by magrande — COMMENTED** [review]
LGTM, closing the PR.

**Pull request closed without merging**